ansible/roles/runner/files/prci.service

```diff
@@ -10,6 +10,9 @@ WorkingDirectory=/root/freeipa-pr-ci
 ExecStart=/bin/bash -c '/root/freeipa-pr-ci/github/prci.py "$(hostname -s)" --config /root/.config/freeipa-pr-ci/config.yml'
 StandardOutput=syslog
 StandardError=syslog
+Restart=on-failure
+RestartSec=10m
+StartLimitIntervalSec=0
```

Comment:
```
Dec 05 10:55:29 .... systemd[1]: [/etc/systemd/system/prci.service:15] Unknown lvalue 'StartLimitIntervalSec' in section 'Service'
```
`StartLimitIntervalSec` belongs to the `[Unit]` section.

Reply:
Ah, that's right.. Thanks for the catch.